### PR TITLE
Fix some UX problems

### LINKED ISSUE #620: Token Swap/Fix some UX problems
- tokenSwapThunks try to map quotes when is null.
- The initial value of inputs should be empty instead 0.
- Don't call the api when value is zero
- When you change between tokens is not asking for new quotes.
- The dropdown should be closed if you click outside
- Max btn isn't with the correct fontWeight and should be have pointer cursor.
- The list of accounts inside of the dropdown should be have pointer cursor

### PR DESCRIPTION
**Pull request by adrifdez**
Closes #620 

### What does this PR does?

Fix some Ux problems that we have found in the flow of token swap

## Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Respect code style and lint
- [ ] Update documentation (if needed)

### CODE REVIEW

**Review by AlbertoElias — DISMISSED** [review]

File: src/apis/token-swap.js [hunk 1]
```diff
@@ -16,11 +16,12 @@ const baseApiUrl = `${process.env.REACT_APP_TOKEN_SWAP_API_URL}`
  * @returns {Array} - Quotes
  */
 function getQuotes (data) {
-  return axios.post(
-      `${baseApiUrl}/quotes`,
-      data,
-      { headers: { 'Content-Type': 'text/plain' } }
-  ).then(res => res.data.quotes)
+  const str = Object.keys(data).reduce(
```
Comment (elias-garcia) [inline src/apis/token-swap.js:19]: You don't need to build the URI manually with axios. This is enough:

```js
return axios.get(`${baseApiUrl}/quotes`, { params: data })
    .then(res => res.data)
```

File: src/views/token-swap/components/swap-button/swap-button.view.jsx [hunk 2]
```diff
@@ -4,20 +4,19 @@ import PropTypes from 'prop-types'
...
 import useSwapButtonStyles from './swap-button.style'
 
-function SwapButton ({ quotes }) {
+function SwapButton ({
+  quotes,
+  selectedQuote
+}) {
   const classes = useSwapButtonStyles()
-  const [timeUntilValid, setTimeUntilValid] = React.useState(30000)
+  const [timeUntilValid, setTimeUntilValid] = React.useState(selectedQuote?.validUntil.getTime())
```
Comment (elias-garcia) [inline src/views/token-swap/components/swap-button/swap-button.view.jsx:12]: Why do we need this state variable? I think that we are only using it inside the effect to set the `countdown` right?

File: src/views/token-swap/components/swap-button/swap-button.view.jsx [hunk 3]
```diff
@@ -28,30 +27,26 @@ function SwapButton ({ quotes }) {
...
 
-  const buttonLabels = {
-    successful: `Swap ${msToTime(timeUntilValid)}`,
-    failure: 'Insufficient liquidity'
-  }
-
   return (
     <div className={classes.root}>
-      {['successful', 'failure'].includes(quotes.status) &&
+      {selectedQuote &&
```
Comment (elias-garcia) [inline src/views/token-swap/components/swap-button/swap-button.view.jsx:32]: I think that this should be `countdown && ...` as we are now using countdown below instead the `selectedQuote` right?

**Review by elias-garcia — CHANGES_REQUESTED** [review]

File: src/views/token-swap/components/swap-button/swap-button.view.jsx (hunk 3 above)
Comment (adrifdez) [reply src/views/token-swap/components/swap-button/swap-button.view.jsx:31]: selectQuote covers more cases

**Review by elias-garcia — APPROVED** [review]

**Pull request merged**